querybook/server/lib/query_analysis/templating.py

```diff
@@ -71,6 +71,10 @@ def get_default_variables():
     }
 
 
+def get_global_variables(jinja_env):
```

Comment:
just wondering, what is the shape returned, is it List[str]?

Reply:
Yup - a List[str]. Added a docstring to clarify